orchard-provider-aws/src/main/resources/reference.conf

```diff
@@ -1,5 +1,13 @@
 com.salesforce.mce.orchard {
     io.aws {
         logging.uri = ${?ORCHARD_AWS_LOGGING_URI}
+        static.credentials {
+            accessKeyId = "changeme"
+            accessKeyId = ${?ORCHARD_AWS_ACCESS_KEY_ID}
+            secretKey = "changeme"
+            secretKey = ${?ORCHARD_AWS_SECRET_KEY}
+            region = "changeme"
+            region = ${?ORCHARD_AWS_REGION}
```

Comment:
I feel like `region` should be outside of credentials

Reply:
@realstraw let me put `region` to be outside of credentials